src/core/components/courses/ProfileOrganization.vue

```diff
@@ -537,7 +537,14 @@ export default {
         this.v$.tempInterlocutorId.$touch();
         if (this.v$.tempInterlocutorId.$error) return NotifyWarning('Champ(s) invalide(s)');
 
-        await Courses.update(this.profileId, { salesRepresentative: this.tempInterlocutorId });
+        const payload = {
+          salesRepresentative: this.tempInterlocutorId,
+          ...(get(this.course, 'contact._id') &&
```

Comment:
Je pense que tu peux retirer `get(this.course, 'contact._id')` car si l'id n'existe pas la deuxième condition est fausse

Reply:
sauf si l'interlocuteur n'existe pas non plus et du coup là elle est vrai et ça met automatiquement l'interlocuteur qu'on ajoute comme contact